app/controllers/application_controller.rb

```diff
@@ -1,2 +1,8 @@
 class ApplicationController < ActionController::API
+  include DeviseTokenAuth::Concerns::SetUserByToken
+
+  def error_message(johan)
+    error_message = johan.full_messages.to_sentence
+    render json: { message: error_message }, status: 422
```

Comment:
I like you Johan but why are your name in the controller? Is this a trick? 😉

Reply:
fixed!